B/booster/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "booster"
+version = v"0.1.2"
+
+# Collection of sources required to build SFML
+sources = [
+    GitSource("https://github.com/evolbioinfo/booster.git", "a5f5044b192db7ab5572f44ef46abaea007f5ce9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/booster
+install_license ${WORKSPACE}/srcdir/booster/LICENSE
+
+# compile booster
+cd ${WORKSPACE}/srcdir/booster/src
+make -j${nproc}
+
+# move to bindir
+install -Dvm 755 "booster" "${bindir}/booster${exeext}"
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("booster", :booster),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency("OpenMPI_jll", v"4.1.2"),
```

Comment:
Is MPI used at all?  Doesn't look like it?

Reply:
The binary needs the potential to be used with MPI, i.e. the binary has an argument for number of cpu's to use, and needs to be compiled with the MPI flags available.

if there is a best practice to get this effect I am open to it.

Reply:
I don't see any reference at all to MPI in the code. Are you sure you aren't confused with OpenMP, which, as confusing as it is, it's something completely different?